onnxruntime/contrib_ops/cpu/quantization/qlinear_global_average_pool.cc

```diff
@@ -55,6 +55,37 @@ Status ComputeQLinearGlobalAvgPool(
   return Status::OK();
 }
 
+//GCC's unexplained behavior: GCC wouldn't generate corresponding symbols versus function instances below 
```

Comment:
nit; Android is misspelt. Can you also run clang-format so the comment indentation is consistent?

Reply:
Done